### PR TITLE
chore: add volunteer samyabrata-maji

### PR DESCRIPTION
**Pull request by samyabrata-maji**

### CODE REVIEW

File: data/users/samyabrata-maji.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "name": "Samyabrata Maji",
+    "bio": "Frontend Engineer | Designer",
+    "image": "https://avatars.githubusercontent.com/u/116789799?v=4",
+    "categories": [
+        {
+            "slug": "programming | web-development",
```
Comment (codewithvoid) [inline data/users/samyabrata-maji.json:7]: looking at the topics..just programming is fine
```json
"slug": "programming"
```

File: data/users/samyabrata-maji.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+    "name": "Samyabrata Maji",
+    "bio": "Frontend Engineer | Designer",
```
Comment (codewithvoid) [inline data/users/samyabrata-maji.json:3]: You can just mention one title for clarity. Frontend engineer/developer preferably.

**Review by codewithvoid — CHANGES_REQUESTED** [review]
Thanks for volunteering @samyabrata-maji. Left few comments.

File: data/users/samyabrata-maji.json (hunk 1 above)
Comment (samyabrata-maji) [reply data/users/samyabrata-maji.json:7]: I'll go with web-development

File: data/users/samyabrata-maji.json (hunk 2 above)
Comment (samyabrata-maji) [reply data/users/samyabrata-maji.json:3]: ok I'll implement the changes

File: data/users/samyabrata-maji.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+    "name": "Samyabrata Maji",
+    "bio": "Frontend Engineer",
+    "image": "https://avatars.githubusercontent.com/u/116789799?v=4",
+    "categories": [
+        {
+            "slug": "web-development",
+            "topics": [
```
Comment (codewithvoid) [inline data/users/samyabrata-maji.json:8]: limit these to max 2-3

File: data/users/samyabrata-maji.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
+{
+    "name": "Samyabrata Maji",
+    "bio": "Frontend Engineer",
+    "image": "https://avatars.githubusercontent.com/u/116789799?v=4",
+    "categories": [
+        {
+            "slug": "web-development",
```
Comment (codewithvoid) [inline data/users/samyabrata-maji.json:7]: i would suggest change this to programming as topics contain programming languages.

you can add separate entry for web development mentioning web frameworks and technologies you are proficient (keep them max 2-3 too)

Comment (samyabrata-maji) [reply data/users/samyabrata-maji.json:7]: ok I'll  add `programming` slug then

File: data/users/samyabrata-maji.json (hunk 3 above)
Comment (samyabrata-maji) [reply data/users/samyabrata-maji.json:8]: sure

**Review by codewithvoid — APPROVED** [review]
Thanks for making quick changes. Welcome!

**Pull request merged**